theano/tensor/nnet/nnet.py

```diff
@@ -2448,6 +2448,29 @@ def elu(x, alpha=1):
     return tensor.switch(x > 0, x, alpha * tensor.expm1(x))
 
 
+def selu(x):
```

Comment:
Add that in the doc in the file doc/library/tensor/nnet/nnet.txt. Add elu at the same time.

After that, I think the PR will be ready to merge.

thanks